proto/php/example.php

```diff
@@ -25,9 +25,9 @@ public function index(string $value): Response
 
 
         [$reply, $status] = $client->TransformText(
-            new Transformation(
+            new TransformTextRequest(
                 [
-                    'transformer' => Transformer::UPPERCASE,
+                    'transformer' => Transformer::TRANSFORMER_UPPERCASE_UNSPECIFIED,
```

Comment:
did you regen the php stubs as well ? cause here we need the UPPERCASE transformer

Reply:
sorry forget to update the php part will do in a sc